src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -357,10 +357,14 @@
 			"  # +------------------------------------------------------+ #",
 			"  ############################################################",
 			""),
+	GTOWN_SETTINGS_MENU_LISTING(

```

Comment:
Lets the fancy GUI thing be turned off.

Reply:
Not sure if we want the fancy GUI just yet